src/components/spec-document/try-it/TryIt.vue

```diff
@@ -110,7 +110,7 @@ const props = defineProps({
 
 })
 
-const authComponentRef = useTemplateRef<InstanceType<typeof TryItAuth>>('authComponentTemplateRef')
+const authComponentRef = useTemplateRef<InstanceType<typeof TryItAuth>>('auth2ComponentTemplateRef')
```

Comment:
```suggestion
const authComponentRef = useTemplateRef('auth2ComponentTemplateRef')
```

We should also get rid of passing the generics here, which prevented TypeScript to catch the wrong ref name.

<img width="2511" height="766" alt="Image" src="https://github.com/user-attachments/assets/90d0bb18-6049-479f-a8c7-fd29e49b215f" />